v2/routes/commonAPI.js

```diff
@@ -584,6 +584,15 @@ function commonAPI(db, io, isInstructor) {
 				throw 'chat modify error';
 			}
 
+			if (req.body != slide.pages[+req.body.pageNum - 1].questions[req.body.qid].chats[req.body.cid].body) {
```

Comment:
should be req.body.body
also, === is preferred

Reply:
Also, if you want to do this check, why not move it further up so we can skip the db write as well :) Just send a 200 back and return

Reply:
I think even better, you should just do the check in the client, save some work for the server